wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -116,6 +123,7 @@ function PasswordPrompt() {
                 defaultValue=""
                 name="passwordConfirm"
                 placeholder="Confirm your password"
```

Comment:
We should also use I18n for placeholders. 😇

Reply:
true! there is no i18n in wails yet. I was thinking to add it with @maxime-carabina tomorrow

Reply:
Ok! Fair enough! We can create a MR only for that. Good for me!